src/Ryujinx.Tests/Cpu/CpuTestSimdCvt32.cs

```diff
@@ -511,6 +511,44 @@ private static IEnumerable<ulong> _1D_F_()
 
             CompareAgainstUnicorn();
         }
+
+        [Test, Pairwise, Description("VRINTR.F<size> <Sd>, <Sm>")]
+        public void Vrintr([Values(0u, 1u)] uint rd,
+                           [Values(0u, 1u)] uint rm,
+                           [Values(2u, 3u)] uint size,
+                           [ValueSource(nameof(_1D_F_))] ulong s0,
+                           [ValueSource(nameof(_1D_F_))] ulong s1,
+                           [ValueSource(nameof(_1D_F_))] ulong s2,
+                           [Values(RMode.Rn, RMode.Rm, RMode.Rp)] RMode rMode)
+        {
+            uint opcode = 0xEB60A40;
```

Comment:
```suggestion
            uint opcode = 0xEEB60A40;
```
The instruction should have condition set to "Always", otherwise it will not actually execute.